jgiven-testng/build.gradle

```diff
@@ -5,10 +5,12 @@ description = "Module to write JGiven tests with TestNG"
 
 
 dependencies {
-    implementation project(':jgiven-core')  //TODO: might be api
-    implementation project(':jgiven-html5-report') //TODO: might be api
-    api "org.testng:testng:$testngVersion"
+    api project(':jgiven-core')
+    implementation project(':jgiven-html5-report')
```

Comment:
Just for understanding: what is the reason to keep this as "implementation" instead of changing to "api"?

Reply:
because usually the test report infrastructure comes in via the plugins